pysph/tools/gen_geom_surf.py

```diff
@@ -0,0 +1,121 @@
+'''
```

Comment:
I do not like the name of the module at all.  How about just `mesh.py`?  The function can then be `msh2pts` or `mesh2surf` or `mesh2points`?